.github/workflows/stale-pr.yaml

```
@@ -15,12 +15,12 @@ jobs:
       - uses: actions/stale@5f858e3efba33a5ca4407a664cc011ad407f2008 # v10.1.0
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
-          stale-pr-message: 'This PR was marked stale due to lack of activity. It will be closed in 7 days.'
+          stale-pr-message: 'This PR was marked stale. It will be closed in 30 days without additional activity.'
           close-pr-message: 'Closed as inactive. Feel free to reopen if this PR is still being worked on.'
           exempt-pr-labels: 'release:after-ga'
           # opt out of defaults to avoid marking issues as stale
           days-before-stale: -1
           days-before-close: -1
           # overrides the above only for pull requests
-          days-before-pr-stale: 7
-          days-before-pr-close: 7
+          days-before-pr-stale: 90
+          days-before-pr-close: 30
```

Given the round trip times we sometimes have in the repo, 14 days in total might indeed be too little, but going from 14 to 120 days (or 4 months) without any activity at all might be too much as well.

How about 14 + 14 days (~ 1 month without activity)?


```suggestion
          stale-pr-message: 'This PR was marked stale. It will be closed in 14 days if there is no activity.'
          close-pr-message: 'Closed as inactive. Feel free to reopen if this PR is still being worked on.'
          exempt-pr-labels: 'release:after-ga'
          # opt out of defaults to avoid marking issues as stale
          days-before-stale: -1
          days-before-close: -1
          # overrides the above only for pull requests
          days-before-pr-stale: 14
          days-before-pr-close: 14
```

---

Agree. Let's double to 14+14 for now.

---

While better than 7, I like the guideline from @breedx-splk that it should be longer than a long European holiday which suggests to me that 14 days is also too short. Perhaps 21 days?

---

I'd prefer to stick with 14+14 and expand if we still see an issue then.  21 days and the PR queue becomes even more unmanagable.  WIth 14+14, that's 28 days you have to mark your PR not stale of have some comment before it closes. I believe that's more than enough time for a normal holiday, and if you're leaving for longer, ask someone to help push your PR while you're out.